composer/trainer/dist_strategy.py

```diff
@@ -132,6 +132,10 @@ def get_torch_dtype(dtype: Union[Precision, str]):
         return torch.float16
     elif dtype in ['bfloat16', 'bfloat', 'torch.bfloat16', 'bf16', 'amp_bf16']:
         return torch.bfloat16
+    elif dtype in ['amp_fp8']:
+        # We use torch.bfloat16 by default for amp_fp8 as there is no
+        # fp8 datatype in PyTorch yet.
+        return torch.bfloat16
```

Comment:
Hi! New to your repo! Fp8 integration sounds super nice! I'm just trying to understand what this line implies? Does it mean that everything is running in bf16 when we specify fp8?

Reply:
Nvm I think I misunderstood how the fp8 system worked I was somehow expecting to play with a specific dtype. Not the context manager. My bad

Reply:
Glad that your confusion is resolved.